### DISCUSSION TITLE
A problem of Ionosphere estimation

### DISCUSSION THREAD

**Discussion by k9891** [groops-devs/groops#245] [Q&A]
Hello developers! Generally, I hope to estimate the STEC parameters directly through groops instead of mapping them into VTEC.  I have turned off both codebias and tecbias parameters  , but it is estimated that STEC has negative values, I'm not sure what's wrong with it.
I would aprreciate for you patience and look forward to your kind reply!
kind regards,
k9891

**Comment by tmayerguerr** [comment]
Hi k9891,

The STEC estimate is a biased estimate, as the values also contain code bias components (the TecBias component). These values can therefore also be negative.

Code bias and TEC can only be separated using additional information such as the use of TEC maps or VTEC mapping.

Best regards
Torsten

**Comment by k9891** [comment]
Hello developers! Thank you for your answer. I know STEC estimate is a biased estimate, which contians the DCB components. In PPP processing,  If the TEC maps or VTEC mapping are not set and the satellite bias is corrected using the .BIA file, whether the output stec in the residual file is a biased estimate of the result? Because in the paper, seeing that stec may involve  ionospheric scintillations.
Best regards
K9891
